fastlane_bot/events/utils.py

```diff
@@ -598,13 +629,16 @@ def get_config(
 
     """
     default_min_profit_gas_token = Decimal(default_min_profit_gas_token)
+    if rpc_url:
+        rpc_url = get_rpc_url(blockchain, tenderly_fork_id, rpc_url)
```

Comment:
Pointless, because at present, if `rpc_url` evaluates to a true result (i.e., it is a string of length larger than zero), then `get_rpc_url` simply returns this value as is. If anything, it should be called `if not rpc_url`.